e2e/app/element/base-element.ts

```diff
@@ -288,7 +289,7 @@ export default class BaseElement extends Container {
    */
   async isCursorNotAllowed(): Promise<boolean> {
     return this.getComputedStyle('cursor').then((cursor) => {
-      return cursor === 'not-allowed';
+      return cursor && cursor === 'not-allowed';
```

Comment:
should this be coerced into a boolean with `!!cursor`?

Also, this may be a bit clearer as:
```javascript
const cursor = await this.getComputedStyle('cursor')
return !!cursor && cursor === 'not-allowed'
```

Reply:
👍🏻